### PR TITLE
feature(integration): GA4 - page call enhancement to take its properties from context

### PR DESCRIPTION
**Pull request by akashrpo**
## Description of the change

> add page call enhancement to take its properties from context.page

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by utsabc — APPROVED** [review]

File: v0/destinations/ga4/data/GA4PageConfig.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "destKey": "title",
+    "sourceKeys": "context.page.title",
+    "required": false
+  },
+  {
+    "destKey": "url",
+    "sourceKeys": "context.page.url",
+    "required": false
+  }
+]
```
Comment (ItsSudip) [inline v0/destinations/ga4/data/GA4PageConfig.json:27]: We can get all of these from properties, why we are not taking it from there?

Comment (utsabc) [reply v0/destinations/ga4/data/GA4PageConfig.json:27]: Ideally @akashrpo lets reference properties first then contextual data (customers can customize the properties values)

Comment (akashrpo) [reply v0/destinations/ga4/data/GA4PageConfig.json:27]: parameters passed in properties will override these values as we are taking custom values as well

Comment (utsabc) [reply v0/destinations/ga4/data/GA4PageConfig.json:27]: Got it make sense

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**